HerPublicWebsite.BusinessLogic/ExternalServices/EmailSending/GovUkNotifyApi.cs

```diff
@@ -22,30 +22,35 @@ public GovUkNotifyApi(IOptions<GovUkNotifyConfiguration> config, ILogger<GovUkNo
             this.logger = logger;
         }
 
-        private EmailNotificationResponse SendEmail(GovUkNotifyEmailModel emailModel)
+        private void SendEmail(GovUkNotifyEmailModel emailModel)
         {
             try
             {
-                var response = client.SendEmail(
+                client.SendEmail(
                     emailModel.EmailAddress,
                     emailModel.TemplateId,
                     emailModel.Personalisation,
                     emailModel.Reference,
                     emailModel.EmailReplyToId);
-                return response;
             }
             catch (NotifyClientException e)
             {
                 if (e.Message.Contains("Not a valid email address"))
                 {
-                    throw new EmailSenderException(EmailSenderExceptionType.InvalidEmailAddress);
+                    logger.LogWarning("GOV.UK Notify could not send to an invalid email address");
+                }
+                else if (e.Message.Contains("send to this recipient using a team-only API key"))
```

Comment:
I haven't looked into it, but I assume there isn't a nicer way of checking the errors than parsing the message?

Reply:
Not obviously - I agree it's a bit unsatisfactory, but at least for now if they change the message it'll just fall through to logging the entire message.